### DISCUSSION TITLE
Is it possible with picocli to create a command-line scripting utility like netsh for windows

### DISCUSSION THREAD

**Discussion by AmibeSkyfy16** [remkop/picocli#1647] [Q&A] [answered]
when we enter netsh in a terminal, we enter in a the program and then we can begin to use the tool with all the commands

**Comment by remkop** [comment, marked as answer]
Yes that is possible.

See the [Picocli Shell JLine2](https://github.com/remkop/picocli/tree/main/picocli-shell-jline2) and [Picocli Shell JLine3](https://github.com/remkop/picocli/tree/main/picocli-shell-jline3) modules.

**Reply by AmibeSkyfy16** [reply]
thanks, perfect
